pyaerocom/aeroval/compute_fairmode_stats.py

```diff
@@ -0,0 +1,98 @@
+"""
+Functions for computing the FAIRMODE statistics
+"""
+
+from math import sqrt, isclose
+from typing import Tuple
+
+
+def _get_RMSU(mean: float, std: float, species: str) -> Tuple[float, float, float, float]:
+    species_values = dict(
+        concno2=dict(
+            UrRV=0.24,
+            RV=200,
+            alpha=0.2,
+        ),
+        vmro3=dict(
+            UrRV=0.18,
+            RV=120,
+            alpha=0.79,
+        ),
+        concpm10=dict(
+            UrRV=0.28,
+            RV=50,
+            alpha=0.13,
+        ),
+        concpm25=dict(
+            UrRV=0.28,
+            RV=25,
+            alpha=0.3,
+        ),
+    )
```

Comment:
If `species_values` is a constant, it  could be defined as a CONSTANT at the top of the module, .e.g.

``` python
SPECIES = dict(
    concno2=dict(UrRV=0.24, RV=200, alpha=0.2),
    vmro3=dict(UrRV=0.18, RV=120, alpha=0.79),
    concpm10=dict(UrRV=0.28, RV=50, alpha=0.13),
    concpm25=dict(UrRV=0.28, RV=25, alpha=0.3),
)
```